src/main/resources/application-dev.yaml

```diff
@@ -8,5 +39,23 @@ spring:
       schema.registry.url: ${KAFKA_BROKERS}
     consumer:
       group-id: srvc01
+  jpa:
+    show-sql: false
+    properties:
+      hibernate:
+        format_sql=false
+    hibernate:
+      ddl-auto: create
+  flyway:
+    enabled: false
+
+logging:
+  config: "classpath:logback-test.xml"
+
+server:
+  port: 8090
+
+FAMILIE_BA_SAK_API_URL: http://localhost:8089/api
```

Comment:
Denne appen kjører på port 8088.

Reply:
8089 mente HS.

Reply:
Gjør det nå ja, gjorde ikke det for noen timer siden :)